### PR TITLE
DUOS-980[risk=no]User Backpopulation

### PR DESCRIPTION
**Pull request by raejohanek**
SCOPE:
- Add institution_id column to dacuser
- Standardize institution names in user_property using the Institution Backpopulation spreadsheet that is in the DUOS shared drive
- Populate dacuser institution_id column by cross referencing each user's now standardized institution names in user property with the institution table and get the id that corresponds with that name
- tested this on a smaller scale in postico and it worked well

ADDRESSES:
https://broadworkbench.atlassian.net/browse/DUOS-980

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/DataBiosphere/consent/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing
- [ ] I've updated Swagger to reflect any API changes

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/changesets/changelog-consent-65.0.xml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+     
+      <addColumn tableName="dacuser">
+        <column name="institution_id" type="bigint"/>
+      </addColumn>
+
+
+      <!--standardize institution names in user_property table so that all institution names exist in the institution table
+       and thus all dacusers can have institution ids assigned to them-->
+      <sql>
+      update user_property AS p SET propertyValue = 'Boehringer Ingelheim' WHERE p.propertyValue = 'Boehringer Ingelheim Pharma GmbH & Co KG';
```
Comment (raejohanek) [inline src/main/resources/changesets/changelog-consent-65.0.xml]: even though & is highlighted red it is not causing issues as a SQL query

Comment (rushtong) [reply src/main/resources/changesets/changelog-consent-65.0.xml]: It is causing issues in the tests though. I think we need to escape it, will investigate.

Comment (raejohanek) [reply src/main/resources/changesets/changelog-consent-65.0.xml]: I see, maybe &amp; instead

Comment (raejohanek) [reply src/main/resources/changesets/changelog-consent-65.0.xml]: woah I did not type & it just turned it into that lol but thank you for bringing this to my attention I wasn't sure why the tests were failing

Comment (rushtong) [reply src/main/resources/changesets/changelog-consent-65.0.xml]: The cleanest (I think) way to make this work is to replace the `&` with its `chr` value and concatenation, i.e., 

```
... WHERE p.propertyValue = 'Boehringer Ingelheim Pharma GmbH ' || chr(38) || ' Co KG';
```

File: src/main/resources/changelog-master.xml [hunk 2]
```diff
@@ -67,4 +67,5 @@
     <include file="changesets/changelog-consent-62.0.xml" relativeToChangelogFile="true"/>
     <include file="changesets/changelog-consent-63.0.xml" relativeToChangelogFile="true"/>
     <include file="changesets/changelog-consent-64.0.xml" relativeToChangelogFile="true"/>
```
Comment (rushtong) [inline src/main/resources/changelog-master.xml:69]: When I check out the PR, these files aren't there. I think this PR needs to be rebased to get https://github.com/DataBiosphere/consent/pull/1002

Comment (rushtong) [reply src/main/resources/changelog-master.xml:69]: Resolved!

File: src/main/resources/changesets/changelog-consent-65.0.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        <column name="institution_id" type="bigint"/>
+      </addColumn>
+
+
+      <!--standardize institution names in user_property table so that all institution names exist in the institution table
+       and thus all dacusers can have institution ids assigned to them-->
+      <sql>
+      update user_property AS p SET propertyValue = 'Boehringer Ingelheim' WHERE p.propertyValue = 'Boehringer Ingelheim Pharma GmbH &amp; Co KG';
+      update user_property AS p SET propertyValue = 'BostonGene' WHERE p.propertyValue = 'BostonGene Corp';
+      update user_property AS p SET propertyValue = 'Brazilian National Cancer Institute (INCA)' WHERE p.propertyValue = 'Brazilian National Cancer Institute';
```
Comment (rushtong) [inline src/main/resources/changesets/changelog-consent-65.0.xml]: nit: upper case the `update` keywords

**Review by rushtong — APPROVED** [review]
One minor nit, then 👍🏽

**Pull request merged**